packages/cli/src/ExternalSecrets/ExternalSecrets.controller.ee.ts

```diff
@@ -4,19 +4,22 @@ import { NotFoundError } from '@/ResponseHelper';
 import { Response } from 'express';
 import { Service } from 'typedi';
 import { ProviderNotFoundError, ExternalSecretsService } from './ExternalSecrets.service.ee';
+import { RequireGlobalScope } from '@/decorators/Scopes';
 
 @Service()
-@Authorized(['global', 'owner'])
+@Authorized()
 @RestController('/external-secrets')
 export class ExternalSecretsController {
 	constructor(private readonly secretsService: ExternalSecretsService) {}
 
 	@Get('/providers')
+	@RequireGlobalScope('externalSecretsStore:list')
```

Comment:
should we consider moving auth scopes to existing decorators?
I'm wondering is there is an advantage of having a separate decorator.

Reply:
We could do it as part of the routing decorators but I personally think separate function decorators look neater.